meteor-app/imports/ui/pages/workspace/component.js

```diff
@@ -149,62 +149,60 @@ export default class WorkspacePage extends React.Component {
         </fieldset>
         <fieldset>
           <legend>Map</legend>
+          <ul className="layer-list">
+            {layers.map((layer, layerIndex) => (
+              <li key={layerIndex}>
+                <div>
+                  <input title="Toggle Visibility" type="checkbox" checked={!layer.invisible} data-layer-index={layerIndex} onChange={this._bound_layerVisibilityOnChange} />
+                  <label>{layer.name}</label>
+                </div>
+                <div>
+                  <label>Opacity: </label>
+                  <input type="range" min="0" max="255" step="1" value={layer.opacity * 255} data-layer-index={layerIndex} onChange={this._bound_layerOpacityOnChange} />
+                  <label>{layer.opacity.toFixed(2)}</label>
+                </div>
+              </li>
+            ))}
+          </ul>
           <div className="section_map">
-            <ul className="layer-list">
```

Comment:
You need to be more specific in those comments and what should be changed. Do you mean layer-list being outside section_map instead of inside? It doesn't matter here but in the final layout the slider and the layer list make up one section and the map is a different section.